packages/dashboard/src/customization/propertiesSections/constants.ts

```diff
@@ -72,6 +72,9 @@ export const dropdownConsts = {
       { label: 'Bottom', value: 'bottom' },
     ],
   },
+  legendDisplay: {
```

Comment:
legend display ? is this for showing and not showing legend ? this is a bit confusing for me

Reply:
No, this is not related to showing and not showing legend. 

this is to show Display section in legend
![image](https://github.com/awslabs/iot-app-kit/assets/142866907/c3b8aa9c-5a0f-483c-8691-7fa942246c18)

Ok, I'll make more specific.

Reply:
updated to "legendDisplaySection"